src/store/actions/resource.js

```diff
@@ -346,42 +346,38 @@ export const createResourceAction = (
   };
 
   const insertedH5pResource = await resourceService.h5pToken(data);
-
-  if (!insertedH5pResource.fail) {
-    const resource = insertedH5pResource;
-
-    const createActivityDate = {
-      h5p_content_id: resource.id,
-      playlist_id: playlistId,
-      thumb_url: metadata.thumbUrl,
-      action: 'create',
-      title: metadata.metaContent.metaTitle,
-      type: 'h5p',
-      content: 'place_holder',
-      subject_id:
+  try {
+    if (!insertedH5pResource.fail) {
+      const resource = insertedH5pResource;
+      const createActivityDate = {
+        h5p_content_id: resource.id,
+        playlist_id: playlistId,
+        thumb_url: metadata.thumbUrl,
+        action: 'create',
+        title: metadata.metaContent.metaTitle,
+        type: 'h5p',
+        content: 'place_holder',
+        subject_id:
         metadata.metaContent.metaSubject
         && metadata.metaContent.metaSubject.subject,
-      education_level_id:
+        education_level_id:
         metadata.metaContent.metaEducationalLevels
         && metadata.metaContent.metaEducationalLevels.name,
```

Comment:
Should be indented 2 more spaces

Reply:
client didn't through error on it. seems to be ok then